.circleci/config.yml

```diff
@@ -391,6 +391,27 @@ jobs:
       - store_artifacts:
           path: /go/src/github.com/cosmos/gaia/gaia-windows-res.yml
 
+  dredd:
+    <<: *linux_defaults
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Get Node and test REST implementation against documentation
+          command: |
+            go get github.com/snikch/goodman/cmd/goodman
+            make build
+            make build-contract-tests-hooks
+            export PATH=~/.local/bin:$PATH
+            mkdir -p /go/src/github.com/cosmos/cosmos-sdk/client/lcd/swagger-ui/
+            cp /go/pkg/mod/github.com/cosmos/cosmos-sdk\@v0.28.2-0.20190605232616-5f9c3fdf8895/client/lcd/swagger-ui/swagger.yaml /go/src/github.com/cosmos/cosmos-sdk/client/lcd/swagger-ui/swagger.yaml
```

Comment:
This hardcoded package sum should be gone, someone know a way to access this path from the makefile?